src/main/resources/org/sonatype/nexus/ci/iq/PolicyEvaluationProjectAction/jobMain.groovy

```diff
@@ -17,15 +17,32 @@ import org.sonatype.nexus.ci.iq.PolicyEvaluationHealthAction
 import org.sonatype.nexus.ci.iq.PolicyEvaluationProjectAction
 
 import jenkins.model.Jenkins
+import groovy.json.JsonBuilder
 import lib.JenkinsTagLib
 import lib.LayoutTagLib
 
+// number of last recent builds
+final MAX_BUILDS_TO_GRAPH = 5;
+
 def t = namespace(JenkinsTagLib.class)
 def l = namespace(LayoutTagLib.class)
 
 def projectAction = (PolicyEvaluationProjectAction) it
 def actions = projectAction.getJob().lastCompletedBuild.getActions(PolicyEvaluationHealthAction.class)
 
+def policyEvaluations = projectAction.getJob().getBuilds().stream()
```

Comment:
0 : what happens for the first build on the project/repo with this method call chain?

Reply:
It returns only one instance of `Summary` object.
The user could press the Refresh page button until the first build is in progress then we don't display any Nexus plugin on the page. After the first build is finished you have to refresh the page to show relevant Nexus summary on the page. There is no handler to notify about change build status.